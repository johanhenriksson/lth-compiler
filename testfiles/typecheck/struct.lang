struct test_type { }

int main() {
    test_type struct1; // ok
    not_def struct2; // error
}
