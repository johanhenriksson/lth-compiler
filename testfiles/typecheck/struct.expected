Error at line 5,5: unknown type 'not_def'
